test/python/tensorflow/tests_linux_cpu.txt

```diff
@@ -54,12 +54,12 @@ scan_ops_test.CumsumTest.test1D
 scan_ops_test.CumsumTest.test2D
 scan_ops_test.CumsumTest.test3D
 
-
 #Unable to add a parameter to a function with no parameters!
 argmax_op_test.ArgMaxTest.testInt64
 array_ops_test.StridedSliceGradTest.testInt64Indices
 cast_op_test.CastOpTest.testIntToFloatBoundary
 math_ops_test.AddNTest.testInt
+math_ops_test.ModTest.testFixed
```

Comment:
Is this a new test?

Reply:
I re-enabled this test since we're restricting FloorMod only to float types now.